tests/ui/explicit-tail-calls/higher-ranked-arg.rs

```diff
@@ -0,0 +1,13 @@
+// Regression test for <https://github.com/rust-lang/rust/issues/144826>.
+//@ check-pass
+
+#![feature(explicit_tail_calls)]
+//~^ WARN the feature `explicit_tail_calls` is incomplete
```

Comment:
(while I'm here, maybe a little `#![allow(incomplete_features)]` to remove the unnecessary stderr?!)

Reply:
I personally prefer the warn because it forces it being removed when the feature gets un-incomplete-ified

Reply:
All other tail call tests use `#![expect(incomplete_features)]`